docs/refmans/gui/viselements/generic/progress.md_template

```diff
@@ -3,26 +3,193 @@ A control that can show the progression of some operation.
 
 # Details
 
-TODO
+The progress indicator represents, in a compact way, the ratio of an achievement on a scale from 0
+to 100 percent.<br/>
+It is useful in situations where a lengthy process is underway, and the application needs to reflect
+how much of the task has been completed at any given moment.
 
-# Styling
+The progress indicator represents, in a compact way, the ratio of achievement on a scale from 0 to
+100 percent.<br/>
+It is useful in situations where a lengthy process is underway, and the application needs to reflect
+how much of the task has been completed at any given moment.
+
+The progress is represented as a ring, or an horizontal bar if the [*linear*](#p-linear) property is
+set to True.<br/>
+The amount of the shape that is filled correlates with the percentage. The unfilled portion of the
+circle or the bar represents the remaining progress.
 
-All the date controls are generated with the "taipy-progress" CSS class. You can use this class
-name to select the date selectors on your page and apply style.
+The percentage value can be displayed by setting the [*show_value*](#p-show_value) property is set
+to True.<br/>
+The control can also display a title (using the [*title*](#p-title)) that can be displayed at
+different locations, depending on the setting of the [*title_anchor*](#p-title_anchor) property.
 
 # Usage
 
-## Circular Progress
+## Circular progress indicator {data-source="gui:doc/examples/controls/progress-simple.py"}
+
+The simplest and most compact use case for the progress control is a circular progress
+indicator.<br/>
+How much of the indicator's ring is filled reflects the percentage of achieved progress.
```

Comment:
should the indeterminate mode be mentioned ?

Reply:
Right
Thx

Reply:
Done.